datahub-web-react/src/app/entity/group/GroupMembers.tsx

```diff
@@ -137,12 +137,13 @@ export default function GroupMembers({ urn, pageSize, isExternalGroup, onChangeM
         }, 3000);
     };
 
-    const onRemoveMember = (memberUrn: string) => {
+    const onRemoveMember = (memberEntity: CorpUser) => {
+        const memberName = memberEntity?.properties?.displayName || undefined;
```

Comment:
please use entityRegistry.getDisplayName(EntityType.CorpUser, memberEntity)

Reply:
@jjoyce0510 - Logic updated to pull display name using above command. Thanks!